help/destinations/catalog/advertising/magnite-batch.md

```diff
@@ -161,19 +164,19 @@ On the "Configure a filename and export schedule for each audience" screen, you
 >
 > A Mapping ID or "NONE" is required for this destination.
 >
-> A Mapping ID should be provided when an audience has a pre-existing Segment ID previously known to Magnite Streaming. Otherwise, "NONE" should be used as the Mapping ID.
+> A Mapping ID should be provided when an audience has a pre-existing Segment ID previously known to [!DNL Magnite]. Otherwise, "NONE" should be used as the Mapping ID.
```

Comment:
Avoid using the passive voice. Talk directly to the reader:

"Provide a Mapping ID when your audience has a pre-existing Segment ID previously known to [!DNL Magnite]. Otherwise, enter "NONE" as the Mapping ID."